app/views/sorry/unable.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Request</h1>
+
+<div>
+  <h2>We're sorry...</h2>
+</div>
+
+<div>
+  We're unable to complete your request right now.
+</div>
+
+<div>
+  Please contact the circulation desk in Green Library at greencirc@stanford.edu or 650-723-1493.
```

Comment:
We have this information configured here: https://github.com/sul-dlss/sul-requests/blob/master/config/application.rb#L151-L154